Makefile.js

```diff
@@ -560,7 +560,7 @@ target.mocha = () => {
 target.karma = () => {
     echo("Running unit tests on browsers");
 
-    target.webpack();
+    target.webpack("production");
```

Comment:
Seems like it would be better to test against the production build in our Karma tests.